python/HeaderService/camera_coords.py

```diff
@@ -1,6 +1,10 @@
 import HeaderService
+import math
+
+"""This modules describe the CCD/Camera coordinate systems as described in
```

Comment:
Here and elsewhere, Python files need to have the LSST license header, and docstrings should conform to the standards in https://developer.lsst.io/python/numpydoc.html

Reply:
If understand this correctly, the string is correctly formatted, but I'm missing the License? If that is the case, I'd like to create a new separate ticket to add the License to all relevant files in both salpytools and headerService.

Reply:
"This module" not "This modules"

Reply:
Typo had been fixed too. Thanks for being an attentive reader.

Reply:
I've created a ticket to add the LSST license header as a separate task, DM-15861.